src/Microsoft.AspNet.Mvc.Razor.Host/MvcRazorCodeParser.cs

```diff
@@ -127,8 +129,10 @@ protected virtual void InjectDirective()
                                .Value
                                .Substring(typeName.Length);
 
-            Span.CodeGenerator = new InjectParameterGenerator(typeName.Trim(),
-                                                              propertyName.Trim());
+            // ';' is optional
```

Comment:
@yishaigalatzer suggested we should until: https://github.com/aspnet/Mvc/issues/1864 is completed.